public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -190,4 +194,19 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => ({
     min-height: 100%;
     width: 100%;
   `,
+  settingsContent: css`
+    flex-grow: 1;
+    min-width: 0;
+    height: 100%;
+    padding: 32px;
+    max-width: 1100px;
+  `,
+  settingsContentFullWidth: css`
+    flex-grow: 1;
+    height: 100%;
```

Comment:
the height does not seem to do anything :thinking: -- I needed to add a `calc(100vh - 250px)`, any better ideas?